src/resolvers/exotics/hosted-git-resolver.js

```diff
@@ -71,6 +71,11 @@ export default class HostedGitResolver extends ExoticResolver {
     throw new Error('Not implemented');
   }
 
+  static getGitSSH(exploded: ExplodedFragment): string {
+    exploded;
+    throw new Error('Not implemented');
```

Comment:
That is what it does on all the methods of HostedGitResolver (src/resolvers/exotics/hosted-git-resolver.js) https://github.com/ramasilveyra/yarn/blob/efd9801987ef395a6a5ed30b3bc5e0cef94326b5/src/resolvers/exotics/hosted-git-resolver.js#L58-L84
I think that is for dev assurance when you extend the class HostedGitResolver.